tap_github/streams.py

```diff
@@ -231,6 +236,10 @@ def http_headers(self) -> dict:
         headers["Accept"] = "application/vnd.github.squirrel-girl-preview"
         return headers
 
+    def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
+        row["type"] = "pull_request" if row["pull_request"] else "issue"
```

Comment:
Looks like this is failing (missing?) for regular issues. What do you think of?:

```suggestion
        row["type"] = "pull_request" if "pull_request" in row else "issue"
```